app/electron/main.js

```diff
@@ -205,7 +212,7 @@ app.on('quit', function () {
       intentionalQuit = true;
       log.info('stopping server process...');
       serverProcess.stdin.pause();
-      // TODO: should we try and end the process a bit more gracefully?
+      // @todo: should we try and end the process a bit more gracefully?
```

Comment:
This doesn't belong in this commit. Please add it to the commit that introduced the todo.

Reply:
I squished the commits into one now. Can you please check it again?